.github/workflows/rdev-update-for-pr.yml

```diff
@@ -0,0 +1,105 @@
+name: Create/Update rdev for PR
+
+on:
+  pull_request:
+    branches-ignore:
+      - prod
```

Comment:
Instead of this, should we consider
```
branches:
  - main
 ```
 ?

Reply:
i.e., do we want to spin up an rdev for scenarios where someone creates a (minor) PR against someone else's feature branch?

Reply:
(probably not; we probably want to create one every time... but just a thought)

Reply:
most of our PRs are against main. If we ignore main then most PRs will not spin up an Rdev. The goal is to spin up and run tests on PRs before they get merge into main. I put prod in this list because we usually create PR to merge changes into prod from main. In this case we don't need to spin up an rdev because main is already associated with the dev environment. I had staging in this list be removed it, since that branch will be going away soon.

Reply:
No, my suggestion of
```
branches:
  - main
```
would spin up an rdev for all PRs against main, but not for any PRs against other branches such as prod or a feature branch

Reply:
`branches` vs `branches-ignore`

Reply:
If we did that, then if there was a feature branch it would lack rdevs and testing.